packages/core/README.md

```diff
@@ -228,12 +229,40 @@ Craft.js is made with :heart: by these wonderful people ([emoji key](https://all
 
 This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind are welcome!
 
+## Contributors
 
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+<a href="https://github.com/prevwong/craft.js/graphs/contributors"><img src="https://opencollective.com/craftjs/contributors.svg?width=890&button=false" /></a>
+
```

Comment:
I don't think we need this, since we already have the all-contributors section above